src/redux/index.js

```diff
@@ -0,0 +1,12 @@
+
+import { combineReducers } from 'redux';
+import usersReducer from './users';
+import eventsReducer from './events'
+
+const reducer = combineReducers({
+  users: usersReducer,
```

Comment:
make another 2 states for "user" and "event" so that you dont have state.users.users and state.users.user